tests/src/JIT/Performance/CodeQuality/HWIntrinsic/X86/PacketTracer/Program.cs

```diff
@@ -0,0 +1,167 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+//
+
+using System;
+using System.Diagnostics;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Collections.Concurrent;
+using System.Runtime.Intrinsics.X86;
+//using Microsoft.Xunit.Performance;
+
+//[assembly: OptimizeForBenchmarks]
+
+class Program
+{
+#if DEBUG
+
+    private const int RunningTime = 200;
+    private const int Width = 248;
+    private const int Height = 248;
+    private const int Iterations = 1;
+    private const int MaxIterations = 1000;
+
+#else
+
+    private const int RunningTime = 1000;
+    private const int Width = 248;
+    private const int Height = 248;
+    private const int Iterations = 7;
+    private const int MaxIterations = 1000;
+
+#endif
+
+    private double _framesPerSecond;
+    private bool _parallel;
+    private bool _showThreads;
+    private static int _width, _height;
+    private int _degreeOfParallelism = Environment.ProcessorCount;
+    private int _frames;
+    private CancellationTokenSource _cancellation;
+    private ObjectPool<int[]> _freeBuffers;
+
+    public Program()
+    {
+        _width = Width;
+        _height = Height;
+        _parallel = false;
+        _showThreads = false;
+        _freeBuffers = new ObjectPool<int[]>(() => new int[_width * 3 * _height]);
+    }
+
+    static unsafe int Main(string[] args)
+    {
+        if (Avx2.IsSupported)
+        {
+            var r = new Program();
+            //r.RenderTo("./pic.ppm");
```

Comment:
commented out code?

Reply:
This method is just for debugging and output the picture, we do not need it in the benchmark.

Reply:
A comment indicating that is why it is commented out would be nice 😄